setup.py

```diff
@@ -158,8 +158,12 @@ def run(self):
             # configure
             cmake_args = [
                 CMAKE,
-                f"-DPYTHON_INCLUDE_DIR={sysconfig.get_path('include')}",
-                f"-DPYTHON_EXECUTABLE={sys.executable}",
+                f"-DPython_INCLUDE_DIRS={sysconfig.get_path('include')}",
+                (
+                    f"-DPYTHON_EXECUTABLE={sys.executable}"
+                    if WINDOWS
+                    else f"-DPYTHON_EXECUTABLE={os.path.join(sysconfig.get_path('include'), '..', '..', 'bin', 'python3')}"
```

Comment:
Is this robust?

Reply:
sys.executable may return a wrong path, as discussed in https://github.com/python/cpython/issues/84399

Reply:
I see. Could you add this as a comment?

Reply:
Added